src/Integration.Vsix/Suppression/DelegateInjector.cs

```diff
@@ -44,16 +43,21 @@ namespace SonarLint.VisualStudio.Integration.Vsix.Suppression
     internal sealed class DelegateInjector : IDisposable
     {
         private readonly Func<Diagnostic, bool> suppressionFunction;
+        private readonly IServiceProvider serviceProvider;
 
-        public DelegateInjector(Func<Diagnostic, bool> suppressionFunction)
+        public DelegateInjector(Func<Diagnostic, bool> suppressionFunction, IServiceProvider serviceProvider)
```

Comment:
Writing to the output window as requested by @Evangelink means we need a service provider.